internal/app/machined/internal/server/v1alpha1/v1alpha1_server.go

```diff
@@ -778,6 +774,7 @@ func (s *Server) List(req *machine.ListRequest, obj machine.MachineService_ListS
 }
 
 // DiskUsage implements the machine.MachineServer interface.
+// nolint:cyclop
```

Comment:
```suggestion
//nolint:cyclop
```

Reply:
we also need a linter for that whitespace check :D That's a reflex...